packages/cubejs-questdb-driver/src/QuestQuery.ts

```diff
@@ -87,6 +87,16 @@ export class QuestQuery extends BaseQuery {
       .join(' AND ');
   }
 
+  public renderSqlMeasure(name: string, evaluateSql: string, symbol: any, cubeName: string, parentMeasure: string): string {
+    // QuestDB doesn't support COUNT(column_name) syntax.
```

Comment:
Nit. IIRC COUNT(column_name) counts non-NULL values, not 100% equivalent to COUNT(*)

Reply:
That's something QuestDB users have to keep in mind when defining their cubes. I don't think we can do anything about it on the driver level.